crates/core/Cargo.toml

```diff
@@ -85,10 +85,11 @@ tracing.workspace = true
 url.workspace = true
 urlencoding.workspace = true
 uuid.workspace = true
-wasmer-middlewares.workspace = true
-wasmer-types.workspace = true
-wasmer-vm.workspace = true
-wasmer.workspace = true
+# wasmer-middlewares.workspace = true
+# wasmer-types.workspace = true
+# wasmer-vm.workspace = true
+# wasmer.workspace = true
```

Comment:
Please remove these. Also, if we no longer depend on Wasmer at all, please remove the deps from the workspace. If we do still depend on Wasmer somehow, I'd like a comment in the workspace Cargo.toml describing what we use it for.